packages/host/src/components/Login/LoginLayout.tsx

```diff
@@ -47,10 +47,16 @@ export const LoginLayout = ({
           <Typography
             sx={{
               color: theme => theme.typography.login.color,
-              fontSize: '64px',
+              fontSize: {
+                xs: '20px',
+                sm: '32px',
+                md: '38px',
+                lg: '48px',
+                xl: '64px',
+              },
```

Comment:
The heading font is now a lot smaller on my "normal" view than it was before -- is that intentional?

Previous:
<img width="1464" alt="Screen Shot 2022-04-06 at 8 17 04 PM" src="https://user-images.githubusercontent.com/5456533/161929636-c93aebaa-6b21-4aae-9aff-1a5a3009082f.png">

Now:
<img width="1472" alt="Screen Shot 2022-04-06 at 8 16 47 PM" src="https://user-images.githubusercontent.com/5456533/161929660-798a3f79-d8fc-4693-ac5b-e9a372c6f7df.png">

This seems fine on my screen at all widths:
```
fontSize: {
                xs: '38px',
                sm: '48px',
                md: '64px',
                lg: '64px',
                xl: '64px',
              },
```

Reply:
ah. thanks - no, not intentional.